scenarios/throttling/rules.drl

```diff
@@ -0,0 +1,38 @@
+import io.github.hephaestusmetrics.model.metrics.Metric;
+import java.util.List
+import org.k8loud.executor.actions.openstack.IpThrottleAction
+
+global org.k8loud.executor.model.ActionList actions;
```

Comment:
Won't work in the latest themis version, we got rid off `actions`.
Currently the globals section looks like
```
global org.k8loud.executor.drools.UsableServices usableServices;
global org.k8loud.executor.drools.CronChecker cronChecker;
```